src/anm/player.js

```diff
@@ -1243,15 +1240,8 @@ Player.prototype._drawEmpty = function() {
     ctx.save();
 
     var ratio = engine.PX_RATIO;
-    // FIXME: somehow scaling context by ratio here makes all look bad
-
-    // background
     ctx.fillStyle = Player.EMPTY_BG;
     ctx.fillRect(0, 0, w * ratio, h * ratio);
-    ctx.strokeStyle = Player.EMPTY_STROKE;
-    ctx.lineWidth = Player.EMPTY_STROKE_WIDTH;
-    ctx.strokeRect(0, 0, w * ratio, h * ratio);
-
```

Comment:
Completely forgot about that and, honestly, I also forgot why it's here at all, thank you!